buttercup-compat.el

```diff
@@ -20,7 +20,7 @@
 
 ;;; Commentary:
 
-;; This file provides compatibility definitions for buttercup. These
+;; This file provides compatibility definitions for buttercup.  These
```

Comment:
Why do you add double spaces after a period? Does Emacs complain about that somewhere?

Reply:
done